app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Invitation Controller
+class InvitationsController < ApplicationController
+  # skip_before_action :confirmable, only: :create
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    @user.password = SecureRandom.base64(10)
```

Comment:
use alphanumeric rather than base64

Reply:
Because of some password validation like: special char, numeric value etc, I used it